gemrb/plugins/BIFImporter/BIFImporter.cpp

```diff
@@ -153,6 +154,63 @@ int BIFImporter::OpenArchive(const char* path)
 	return GEM_OK;
 }
 
+#else
```

Comment:
This is a bad idea, but we have a better solution in the works. Let's get back to this as we merge the Vita port fixes, you'll see it will become much cleaner.

It's also not clear why you don't skip all the readonly_mmap stuff then.

Reply:
I didn't skip the readonly_mmap stuff because it is a possibility. And at start it will works in this way but since it is really slow i've reverted to the classic file implementation. But if you tell me that you are working on something else i'll wait. No problem

Reply:
The vita stuff has been merged. When you rebase your branch on current master, the need for this should automatically go away if you also remove your mmap stub, since then the code will fall back to the old behaviour.